ui/src/main/java/edu/wpi/grip/ui/MainWindowController.java

```diff
@@ -236,4 +246,25 @@ public void deploy() {
         dialog.setResizable(true);
         dialog.showAndWait();
     }
+
+    public void generate(ActionEvent actionEvent) {
+        ImageView graphic = new ImageView(new Image("/edu/wpi/grip/ui/icons/settings.png"));
+        graphic.setFitWidth(DPIUtility.SMALL_ICON_SIZE);
+        graphic.setFitHeight(DPIUtility.SMALL_ICON_SIZE);
+    	ChoiceDialog<Language> dialog = new ChoiceDialog(Language.JAVA, Language.JAVA, Language.CPP, Language.PYTHON);
+		dialog.setTitle("Language");
+		dialog.setGraphic(graphic);
+		dialog.setHeaderText("Export Language");
+        dialog.getDialogPane().styleProperty().bind(root.styleProperty());
+        dialog.getDialogPane().getStylesheets().setAll(root.getStylesheets());
```

Comment:
I kind of wish instead of just generating the file, it would display a preview of the file it is going to generate first.  (Take a look at scene builder for java fx)